### PR TITLE
Issue #46 fixed

### PR DESCRIPTION
**Pull request by edouardalvescamilo**
Added a click listener on "menu_top_left" in order to remove the class "scrolledleft" from "welcome".

### CODE REVIEW

**Comment by Yoric** [comment]
That's probably good, but I can't accept the patch because of 54e9658 and 0de2a3a. Can you remove or squish these commits before proceeding?

**Comment by Yoric** [comment]
I still see issue #107 in the pull request, which is surprising.

File: app.js [hunk 1]
```diff
@@ -202,6 +202,10 @@ $("menu_top_right").addEventListener("click", event => {
   }
 });
 
+$("menu_top_left").addEventListener("click", event => {
+	$("welcome").classList.remove("scrolledleft");
```
Comment (Yoric) [inline app.js]: Could you fix indentation?

**Comment by Yoric** [comment]
Could you please rebase again?

**Comment by Yoric** [comment]
There are too many changes in this commit. Can you rebase it?

**Comment by Yoric** [comment]
@AlexManga I'm still waiting for you to squash your changes.

**Comment by AlexManga** [comment]
@edouardalvescamilo the last comment is for you.

**Pull request closed without merging**